.github/workflows/ci.yml

```diff
@@ -131,11 +131,20 @@ jobs:
           toolchain: beta
           override: true
 
-      - name: Install profilers
+      - name: Install Valgrind
```

Comment:
A brief comment about why we build Valgrind ourselves would be useful here.

Reply:
Good idea, I added a comment.